src/tools/perf/api/libperf.h

```diff
@@ -98,6 +98,60 @@ enum {
 };
 
 
+/*
+ *   +------+                                                        +------+
+ *   |HOST 0|----------------------PEER_TX(AM)---------------------->|HOST 1|
+ *   +------+        +---------+                  +---------+        +------+
+ *      |            |DPU DMN 0|                  |DPU DMN 1|            |
+ *      |            +---------+                  +---------+            |
+ *      |                 |                            |                 |
+ *      |---HOST_INIT---->|                            |<----HOST_INIT---|
+ *      |                 |---------PEER_INIT--------->|                 |
+ *      |                 |                            |                 |
+ *      |---HOST_SEND---->|                            |<--HOST_RECV(T)--|
+ *      |                 |--------PEER_TX(AM)-------->|                 |
+ *      |<--HOST_SEND_ACK-|                            |-HOST_RECV_ACK-->|
+ *      |                 |                            |                 |
+ *      |----HOST_FIN---->|                            |<----HOST_FIN----|
+ *
+ * (AM) used only when AM communication is configured
+ * (T) used only when tag matching communication is configured
+ */
+#define UCP_FOREACH_PERF_DAEMON_AM_ID(X) \
+    /* Host->DPU daemon AM initialization message */ \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_INIT,     0) \
```

Comment:
maybe better avoid `HOST` in names, because it is actually a daemon who is being initialized.

Reply:
ok